### DISCUSSION TITLE
Configuration Public IP on azuracast

### DISCUSSION THREAD

**Discussion by blkghost77** [AzuraCast/AzuraCast#4813] [Q&A]
Hello
I configure a Azuracast dedicated server, but at the moment the server is connected to my local network. 
Where can I configure the Public IP address  to make some outside connectivity, before I request to my ISP a dedicated IP address.
Thank you

**Comment by YosefAdPro** [comment]
This is less the part related to AzuraCast.

If your server is behind NAT and has an internal network address, you need to open ports from the Internet (public IP) into your internal network.

The minimum ports you need are 80/443/8000-RANGE etc TCP.
... I think it's a bad idea to have a streaming server at home, unless it's just for testing/dev.
